index.md

```diff
@@ -1492,6 +1492,32 @@ Reply Examples:
       :ircd.stealth.net 302 yournick :syrk=+syrk@millennium.stealth.net
                                       ; Reply for user syrk
 
+### WALLOPS message
+
+         Command: WALLOPS
+      Parameters: <Text to be sent>
+
+The WALLOPS command is used to send a message to all currently connected users who have set the 'w' user mode for themselves.
+The `<Text to be sent>` SHOULD be non-empty.
+
+Servers MAY echo WALLOPS messages to their sender even if they don't have the 'w' user mode.
+
+Servers MAY send WALLOPS only to operators.
+
+Servers may generate it themselves, and MAY allow operators to send them.
+
+Numeric replies:
+
+* [`ERR_NEEDMOREPARAMS`](#errneedmoreparams-461) `(461)`
+* [`ERR_NOPRIVILEGES`](#errnoprivileges-481) `(481)`
+* [`ERR_NOPRIVS`](#errnoprivs-723) `(723)`
+
+Examples:
+
+     :csd.bu.edu WALLOPS :Connect '*.uiuc.edu 6667' from Joshua; WALLOPS
```

Comment:
Maybe consider adding a newline before `;`? It seems like it's used as a comment marker for the example description.